experiments/coredns/pod_delete/pod_delete_ansible_logic.yml

```diff
@@ -0,0 +1,80 @@
+---
+- hosts: localhost
+  connection: local
+
+  vars:
+    c_experiment: "pod-delete"
+    c_duration: "{{ lookup('env','TOTAL_CHAOS_DURATION') }}"
+    c_interval: "{{ lookup('env','CHAOS_INTERVAL') }}"
+    c_force: "{{ lookup('env','FORCE') }}"
+    c_lib: "{{ lookup('env','LIB') }}"
+    c_ns: "{{ lookup('env','CHAOS_NAMESPACE') }}"
+    a_ns: "{{ lookup('env','APP_NAMESPACE') }}"
+    a_label: "{{ lookup('env','APP_LABEL') }}"
+    a_kind: "{{ lookup('env','APP_KIND') }}"
+    kill_count: "{{ lookup('env','KILL_COUNT') }}"
+  
+  tasks:
+    - block:
+
+        - include: pod_delete_ansible_prerequisites.yml
+      
+        - include_vars:
+            file: chaosutil.yml
+
+        ## GENERATE EXP RESULT NAME
+        - block:
+
+            - name: Construct chaos result name (experiment_name)
+              set_fact:
+                c_experiment: "{{ lookup('env','CHAOSENGINE') }}-{{ c_experiment }}"
+
+          when: lookup('env','CHAOSENGINE')    
+
+        ## RECORD START-OF-EXPERIMENT IN LITMUSCHAOS RESULT CR
+        - include_tasks: /utils/runtime/update_chaos_result_resource.yml
+          vars:
+            status: 'SOT'
+            namespace: "litmus"
```

Comment:
Can you change the namespace to `a_ns` because we are creating chaos-result in the app namespace.

Reply:
done